src/blocks/row/styles/editor.scss

```diff
@@ -147,6 +147,121 @@
 		}
 	}
 
+	.wp-block-coblocks-row__inner {
+		&.is-row-placeholder {
+			.block-editor-block-list__layout {
+				align-items: center;
+			}
+			.wp-block-coblocks-column {
+				height: inherit;
+			}
+		}
+
+			.block-editor-block-list__layout {
+				height: 100%;
+				width: 100%;
+
+				> div:not(.block-editor-block-mover) {
+					height: auto;
+					width: 100%;
+					
+					.wp-block-coblocks-column {
+						height: 100% !important;
+						width: 100%;
+
+						&__inner {
+							height: inherit;
+							width: 100%;
+
+							> div {
+								height: inherit;
+
+								> div {
+									height: inherit;
+								}
+							}
+						}
+					}
+				}
+				> .block-list-appender {
+					width: 100%;
+				}
+				.block-editor-block-list__block-edit {
+					height: 100%;
+					> div[data-block] {
+						height: 100%;
+						margin-bottom: 0;
+					}
+				}
+			}
+	}
+	
+	.wp-block-coblocks-row__inner {
```

Comment:
This works, but it seems overly complicated - is this all necessary? Core columns doe it [like this](https://github.com/WordPress/gutenberg/blob/c19d2d908cba695960cf8407bd0b0afc181aa657/packages/block-library/src/columns/style.scss#L76), which is much simpler.

Reply:
Changes made. Good call.